introduction.md

```diff
@@ -0,0 +1 @@
+On my way to becoming a GitHub Master
```

Comment:
echo "On my way to becoming a GitHub Master. I love ExpressJS and JavaScript." > introduction.md
git commit -am "Added Code School Path"
git push